astropy/units/format/generic_parsetab.py

```diff
@@ -1,24 +1,13 @@
-# -*- coding: utf-8 -*-
```

Comment:
One last very annoying thing: could you put the comment on top back?

Reply:
Will do.